definitions.yaml

```diff
@@ -29,6 +29,68 @@ definitions:
           datacenter:
             description: Identifier of the datacenter or cloud provider region, e. g. "eu-west-1"
             type: string
+          availability_zones:
+            type: object
+            description: Number of availability zones that the cluster can be spread across.
+            properties:
+              max:
+                description: Maximum number of availability zones in this region.
+                type: number
+              default:
+                description: Default number of availability zones for a cluster.
+                type: number
+      network:
+        description: Information related to networking in the clusters
+        type: object
+        properties:
+          ranges:
+            description: IP ranges that can be used by clusters
+            type: array
+            items:
+              type: object
+              properties:
+                name:
+                  description: Name of the IP range (eg production or default)
+                  type: string
+                pods:
+                  description: IP ranges for pods
+                  type: object
+                  properties:
+                    cidr_blocks:
+                      description: List of CIDRs that will be used by pods
+                      type: array
+                      items:
+                        type: string
+                services:
+                  description: IP ranges for services in Kubernetes
+                  type: object
+                  properties:
+                    cidr_blocks:
+                      description: List of CIDRs that will be used by services in Kubernetes
+                      type: array
+                      items:
+                        type: string
+                docker:
+                  description: IP ranges for docker
+                  type: object
+                  properties:
+                    cidr_blocks:
+                      description: List of CIDRs that will be used by docker on the nodes
+                      type: array
+                      items:
+                        type: string
```

Comment:
yes you are right and i don't like it here as well. imo we should get rid of it completely but this might not be easy. 

my thinking here is: 
this range is configured on each node. the bridge is there and this ip range can't be routed elsewhere. so this is of interest for the user. we would need to get rid of the docker bridge then. its not used anyway imo (at least on the nodes).

Reply:
we might move to CRI and containerd at some time anyway, then we would not even need to have the daemon there 😉

Reply:
+1 to remove docker ranges now.

Reply:
we might remove this entirely. still discussing here: https://github.com/giantswarm/giantswarm/pull/2202#issuecomment-433038314

any opinion about omiting this and waiting for https://github.com/giantswarm/giantswarm/issues/4490 instead?